sql/api/src/main/scala/org/apache/spark/sql/types/StructField.scala

```diff
@@ -218,9 +224,11 @@ case class StructField(
     metadata.contains(EXISTS_DEFAULT_COLUMN_METADATA_KEY)
   }
 
-  private def getDDLDefault = getCurrentDefaultValue()
-    .map(" DEFAULT " + _)
-    .getOrElse("")
+  private def getDDLDefault =
```

Comment:
shall we keep the original ordering? getCurrentDefaultValue().orElse(getParameterDefault())?